examples/geospatial-python-urban-analysis-with-postgis/src/etl/load/postgis/save_bus_stop.py

```
@@ -0,0 +1,13 @@
+from sqlalchemy import Engine
+
+import geopandas as gpd
+
+def save_bus_stop(gdf: gpd.GeoDataFrame, conn: Engine ) -> bool:
+    """Saves the bus stop GeoDataFrame to PostGIS."""
+    try:
+        gdf.to_postgis("bus_stops",conn, if_exists="replace", index=False)
+        print(f"Successfully saved bus stops to PostGIS.")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)